sample/SampleHost/appsettings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "ConnectionStrings": {
+        "AzureWebJobsDashboard": "UseDevelopmentStorage=true",
+        "AzureWebJobsStorage": "UseDevelopmentStorage=true"
```

Comment:
wary of `UseDevelopmentStorage=true` as we don't fully support storage emulators + I doubt that it would work cross plat